x/genaccounts/legacy/v0_36/migrate.go

```diff
@@ -0,0 +1,138 @@
+// DONTCOVER
+// nolint
+package v0_36
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	v034distr "github.com/cosmos/cosmos-sdk/x/distribution/legacy/v0_34"
+	v034accounts "github.com/cosmos/cosmos-sdk/x/genaccounts/legacy/v0_34"
+	v034staking "github.com/cosmos/cosmos-sdk/x/staking/legacy/v0_34"
+	"github.com/tendermint/tendermint/crypto"
+)
+
+const (
+	notBondedPoolName = "NotBondedTokensPool"
+	bondedPoolName    = "BondedTokensPool"
+	feeCollectorName  = "FeeCollector"
+	mintModuleName    = "mint"
+)
+
+// Migrate accepts exported genesis state from v0.34 and migrates it to v0.36
+// genesis state. All entries are identical except for validator slashing events
+// which now include the period.
+func Migrate(
+	oldGenState v034accounts.GenesisState, fees sdk.Coins, communityPool sdk.DecCoins,
+	vals v034staking.Validators, ubds []v034staking.UnbondingDelegation,
+	valOutRewards []v034distr.ValidatorOutstandingRewardsRecord, bondDenom, distrModuleName, govModuleName string,
+) GenesisState {
+
+	depositedCoinsAccAddr := sdk.AccAddress(crypto.AddressHash([]byte("govDepositedCoins")))
+	burnedDepositCoinsAccAddr := sdk.AccAddress(crypto.AddressHash([]byte("govBurnedDepositCoins")))
+
+	bondedAmt := sdk.ZeroInt()
+	notBondedAmt := sdk.ZeroInt()
+
+	var govCoins sdk.Coins
+	newGenState := make(GenesisState, len(oldGenState)+4) // - 2 gov baseAccs + 6 moduleAccs
```

Comment:
Can we add a more elaborative comments on the number and exact types of accounts?